src/app/components/Login.js

```diff
@@ -384,6 +386,24 @@ class Login extends Component {
                 />
               </div>
 
+              {this.state.type === 'login' ?
+                <div className="login__otp_attempt">
+                  <TextField
+                    fullWidth
+                    name="otp_attempt"
+                    value={this.state.otp_attempt}
+                    className="login__otp_attempt-input"
+                    ref={(i) => { this.inputName = i; }}
```

Comment:
looks like this ref was copied by mistake

Reply:
yes removed it